Makefile

```diff
@@ -328,6 +328,7 @@ test-file:
 .PHONY: clean-githooks
 clean-githooks:
 	-rm -rf ${ROOTDIR}/.git/hooks/* ${ROOTDIR}/.git/hooks.old
+	-git config --local --unset core.hooksPath
```

Comment:
Husky overwrites this value in people's local `.git/config` so this unsets it to return it to its default.